website/docs/cloud-docs/migrate/index.mdx

```diff
@@ -38,17 +38,17 @@ Stop all Terraform operations associated with the state files. This may require
 
    Refer to [The `cloud` Block](/terraform/cli/cloud/settings#the-cloud-block) in the Terraform CLI documentation for additional information. The following example migrates the state associated with the configuration to the `networking` workspace on HCP Terraform:
 
-  ```
-  terraform {
-    cloud {
-      hostname = "app.terraform.io"
-      organization = "my-org"
-      workspaces {
-        tags = ["networking"]
-      }
-    }
-  }
-  ```
+   ```
```

Comment:
should this codeblock have a language? I actually don't remember off the top of my head if `hcl` is supported or not, but I think it's worth it to investigate for the syntax highlighting!